### PR TITLE
Fixes #9744: No server inventory in web interface if /etc/hosts is not correctly configured

### PR DESCRIPTION
**Pull request by peckpeck**
https://www.rudder-project.org/redmine/issues/9744

### CODE REVIEW

File: rudder-webapp/SOURCES/rudder-init [hunk 1]
```diff
@@ -183,6 +183,8 @@ function generate_promises_at_temporary_location() {
 
   cp -r /opt/rudder/share/initial-promises/ ${TMP_DIR}/community
 
+  [ -n "${COMPUTED_ALLOWED_NETWORKS}" ] COMPUTED_ALLOWED_NETWORKS="${COMPUTED_ALLOWED_NETWORKS},"
```
Comment (amousset) [inline rudder-webapp/SOURCES/rudder-init]: Looks like there is a missing "&&"

**Comment by peckpeck** [comment]
Commit modified

**Comment by amousset** [comment]
Why targeting 3.2 and not 3.1?

**Comment by peckpeck** [comment]
PR replaced by https://github.com/Normation/rudder-packages/pull/1149

**Pull request closed without merging**